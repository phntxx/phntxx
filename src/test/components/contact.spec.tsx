import { render } from "@testing-library/react";
import Contact from "../../components/contact";

describe("contact.tsx", () => {
  it("should render component without crashing", () => {
    const { asFragment } = render(<Contact />);
    expect(asFragment).toMatchSnapshot();
  });
});
