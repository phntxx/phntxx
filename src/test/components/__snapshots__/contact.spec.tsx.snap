// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact.tsx should render component without crashing 1`] = `[Function]`;
